source/features/jump-to-conversation-close-event.tsx

```diff
@@ -40,7 +40,8 @@ void features.add(import.meta.url, {
 		pageDetect.isConversation,
 	],
 	include: [
-		pageDetect.isClosedConversation,
+		pageDetect.isClosedIssue,
+		pageDetect.isClosedPR,
```

Comment:
Bumped `github-url-detection` and resolved all breaking changes. I'm separating `isClosedConversation` changes here since this is not a simple search & replace, but reverting [`9e8d7a8` (#5807)](https://github.com/refined-github/refined-github/pull/5807/commits/9e8d7a8128d44ef77c103c2f88f0377ee119c2f1). As mentioned in https://github.com/refined-github/github-url-detection/pull/136#issuecomment-1193295489, the only valid use case is in this feature (`jump-to-conversation-close-event`).